fastagency/models/base.py

```diff
@@ -17,6 +17,9 @@
 
 # abstract class
 class Model(BaseModel, ABC):
+    name: Annotated[
+        str, Field(..., description="The name of the model", pattern="^.+$")
```

Comment:
Why pattern? If you only need it to be non-empty string, you can use `min_length`:

https://docs.pydantic.dev/latest/concepts/fields/#string-constraints

Reply:
In the updated PR, min_length is used instead of pattern